d2renderers/d2svg/d2svg.go

```diff
@@ -220,6 +223,47 @@ func arrowheadMarker(isTarget bool, id string, connection d2target.Connection) s
 				width*0.6, height*7/8,
 			)
 		}
+	case d2target.CfOne, d2target.CfMany, d2target.CfOneRequired, d2target.CfManyRequired:
+		attrs := fmt.Sprintf(`class="connection" stroke="%s" stroke-width="%d" fill="white"`, connection.Stroke, connection.StrokeWidth)
```

Comment:
i think you should fill with connection.Stroke instead of just white. like how `FilledDiamondArrowhead` does

Reply:
oh wait it's supposed to be hollow. in that case just "transparent"

Reply:
The problem with transparent is this:
<img width="247" alt="image" src="https://user-images.githubusercontent.com/4950200/210153077-43845867-0ff5-4c37-9317-01d72534a555.png">

Reply:
oh interesting. the correct thing to do would be to make a mask then (like we do for labels so that connections don't overlap). but i'm okay with doing that later and keeping white for now.